### ISSUE TITLE
Number of records impacted by delete displays 0 records

### ISSUE THREAD

**Issue by n0rthface** [MarkMpn/Sql4Cds#464]
Hi Mark,

I just discovered this minor issue in SQL 4 CDS v9.0.
When I delete any number of records, the number of impacted records in the "Messages" tab equals to 0 in any case after the operation has finished. The number of records is correct for insert and update statements.

![image](https://github.com/MarkMpn/Sql4Cds/assets/14167302/1a6484b8-0c74-4e92-918f-9cc559e88b06)

![image](https://github.com/MarkMpn/Sql4Cds/assets/14167302/adffbe73-1bbd-469d-a558-2d9be611909d)

![image](https://github.com/MarkMpn/Sql4Cds/assets/14167302/cddb48bd-ae2b-4798-bc15-9afab78479ba)

Best regards
Hauke

**Comment by MarkMpn** [comment]
I haven't been able to reproduce this yet. The only expected case for these numbers to be different is if those records are deleted by some other process before SQL 4 CDS can delete them.

Can you check:

1. Are the records still there, or have they been deleted
2. If they have been deleted, is there any chance they were deleted by some other process between you getting the confirmation message and the query completing?
3. If the records are still there, do you get an error message if you try to delete them using some other means?

**Comment by n0rthface** [comment]
I just tested as per your request and the issue still persists for deleted records only.
The records are deleted successfully by issueing the delete statement in SQL 4 CDS. No other process interferes with the delete. When deleting a record from the same table in model driven app, the request returns with status code 204 (no content) in the same way as an update request.
I uninstalled and installed SQL 4 CDS 9.0, but unfortunately there is no change to the behavior.
